buildscripts/docker/scripts/psp_scripts/build_pspdev.sh

```diff
@@ -9,10 +9,8 @@ WORKDIR="$(pwd)"
 cd "${script_root}"
 
 latest_release_url="$(curl -L "https://api.github.com/repos/pspdev/pspdev/releases?per_page=100" \
-    | jq -r '[.[] | select(.prerelease | not)][].assets[].browser_download_url' \
-    | grep 'linux-x64' \
-    | head -n 1)"
+    | jq -r 'first(.[] | select(.prerelease | not).assets[].browser_download_url | contains("linux-x64"))')"
```

Comment:
@bb010g FYI using `contains("linux-x64")` returned `true`.
The correct way was `select(contains("linux-x64"))` :)

Reply:
Sorry, yeah.